services/slam/client_test.go

```diff
@@ -235,44 +235,44 @@ func TestFailingClient(t *testing.T) {
 		conn, err := viamgrpc.Dial(context.Background(), listener.Addr().String(), logger)
 		test.That(t, err, test.ShouldBeNil)
 
-		failingSLAMClient := slam.NewClientFromConn(context.Background(), conn, slam.Named(nameFail).String(), logger)
```

Comment:
Was `slam.Named(nameFail).String()` not needed?

Reply:
When left this as is, I was getting the error in client tests:
`    client_test.go:292: Expected 'error receiving pointcloud chunk: rpc error: code = Unknown desc = resource "rdk:service:slam/rdk:service:slam/maiv" not found' to contain substring 'failure during callback' (but it didn't)!`

Now that we pass the name stored in the client connection into the client requests instead of just the user-inputted name (i.e. `maiv`), we were doubling up on the namespace. The namespace addition was moved to happening in the `client_test.go` fully into `client.go`